hipcheck/src/analysis/score.rs

```diff
@@ -377,14 +396,14 @@ fn wrapped_query(
 			FUZZ_PHASE => db.fuzz_analysis(),
 			REVIEW_PHASE => db.review_analysis(),
 			TYPO_PHASE => db.typo_analysis(),
-			other => Err(hc_error!("Unrecognized legacy analysis '{other}'")),
+			_ => Err(hc_error!("Unrecognized legacy analysis '{other}'")),
```

Comment:
isn't `other` still used in the error message?

Reply:
Oh strange, it appears to compile just fine, and without the `_` it gave a warning. Strange.